test/playwright/utils/audio.ts

```diff
@@ -0,0 +1,72 @@
+import { expect, Locator } from '@playwright/test'
```

Comment:
I kind of hate this module, mostly the implicit assertions that it makes... they feel gross but having to return `Promise<Locator | null>` just means that the expect gets pushed up to the test level instead of a `not.toBeNull()`. Maybe that is better though. My feeling with tests is that the more explicit they are the better, even if they seem repetitive... Do others agree with this? Obviously there's a lot of utility in sharing logic for selecting elements or doing routine actions, but as far as assertions go I think it's nice to have them hoisted up into the test itself :thinking: